packages/fast-components-react-msft/src/button/button.spec.tsx

```diff
@@ -28,6 +29,24 @@ describe("button", (): void => {
 
     const href: string = "https://www.microsoft.com";
 
+    const beforeSlotExample: JSX.Element = (
+        <div
+            className={"slotBefore"}
+            key={"Slot"}
+            slot="before"
+            dangerouslySetInnerHTML={{__html: "Before slot"}}
```

Comment:
Why are you are you dangerously setting inner html?

Reply:
Whoops - this was a hold over from the glyph - fixing.